.github/workflows/build.yml

```diff
@@ -230,38 +231,38 @@ jobs:
   # as we'd otherwise have to build these in CI and commit them back to the PR, which
   # creates all kinds of problems.
   # This failsafe simply aborts anything that has not had charts rebuilt before pushing.
-  # check_charts:
-  #   name: Check if committed Helm charts are up to date
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - name: Install host dependencies
-  #       run: |
-  #         sudo apt-get update
-  #         sudo apt-get install protobuf-compiler krb5-user libkrb5-dev libclang-dev liblzma-dev libssl-dev pkg-config
-  #     - name: Checkout
-  #       uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
-  #       with:
-  #         submodules: recursive
-  #     - name: Set up Helm
-  #       uses: azure/setup-helm@5119fcb9089d432beecbf79bb2c7915207344b78 # v3.5
-  #       with:
-  #         version: v3.6.2
-  #     - name: Set up cargo
-  #       uses: dtolnay/rust-toolchain@1.68.2
-  #     - name: Set up rust-cache
-  #       uses: Swatinem/rust-cache@359a70e43a0bb8a13953b04a90f76428b4959bb6 # v2.2.0
-  #       with:
-  #         key: charts
-  #     - name: Regenerate charts
-  #       run: make regenerate-charts
-  #     - name: Check if committed charts were up to date
-  #       run: git diff --exit-code
-  #     - name: Git Diff showed uncommitted changes
-  #       if: ${{ failure() }}
-  #       uses: actions/github-script@d556feaca394842dc55e4734bf3bb9f685482fa0 # v6
-  #       with:
-  #         script: |
-  #           core.setFailed('Committed charts were not up to date, please regenerate and re-commit!')
+  check_charts:
```

Comment:
It might be worth to cover this as a pre-commit hook.

Reply:
This is pulled in from templating. In isolation I'd agree though.

Reply:
I'd say file an issue on templating about it

Reply:
Okay I will open an issue.

Reply:
Opened an issue here: https://github.com/stackabletech/operator-templating/issues/263